### PR TITLE
IIIFBookReader zoom

### PR DESCRIPTION
**Pull request by gestchild**
Adds image zoom to IIIFBookReader component

feb-04-2019 16-47-37

### CODE REVIEW

File: common/views/components/IIIFBookReader/IIIFBookReader.js [hunk 1]
```diff
@@ -21,7 +22,14 @@ const IIIFBookReader = ({
             .canvases
             .slice(0, 15)
             .map(canvas => {
-              return (<div key={canvas.thumbnail['@id']} ><img src={canvas.images[0].resource['@id']} /></div>);
+              return (<div key={canvas.thumbnail['@id']} style={{position: 'relative', 'marginBottom': 30}}>
+                <ImageViewer
```
Comment (jamesgorrie) [inline common/views/components/IIIFBookReader/IIIFBookReader.js]: Are we loading in loads of DOM here?
Just asking as we are actually starting to get quite heavy.

File: common/views/components/IIIFBookReader/IIIFBookReader.js [hunk 2]
```diff
@@ -21,7 +22,14 @@ const IIIFBookReader = ({
...
+                  contentUrl={canvas.images[0].resource['@id']}
+                  infoUrl={`${canvas.images[0].resource.service['@id']}/info.json`}
+                  id={canvas.images[0].resource['@id']}
+                  width={800}
+                  trackTitle='test'/>
```
Comment (jamesgorrie) [inline common/views/components/IIIFBookReader/IIIFBookReader.js]: ?

File: common/views/components/ImageViewer/ImageViewerImage.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
 // @flow
 import {Component} from 'react';
-import {convertImageUri, convertIiifUriToInfoUri} from '../../../utils/convert-image-uri';
 import {spacing} from '../../../utils/classnames';
 import openseadragon from 'openseadragon';
+import fetch from 'isomorphic-unfetch';
```
Comment (jamesgorrie) [inline common/views/components/ImageViewer/ImageViewerImage.js]: This is from before, but I think the convention is to have libs included first.

File: common/views/components/IIIFBookReader/IIIFBookReader.js (hunk 1 above)
Comment (gestchild) [reply common/views/components/IIIFBookReader/IIIFBookReader.js]: Not much no. Might be worth thinking about how we measure performance of this though

**Pull request merged**